R/method-dispatch.R

```diff
@@ -11,7 +11,8 @@ method_lookup_error <- function(name, args, signatures) {
     msg <- sprintf("Can't find method for generic `%s(%s)`:\n%s", name, arg_names, types)
   }
 
-  stop(msg, call. = FALSE)
+  cnd <- errorCondition(message = msg, class = c("methodNotFound", "error"))
```

Comment:
```suggestion
  cnd <- errorCondition(message = msg, class = c("errorMethodNotFound", "error"))
```

I don't have super strong feelings about it, but in vctrs/rlang we typically include the word `error` in the class name, and we also typically include some package/org related prefix like `vctrs_error_*` or `rlib_error_package_not_found`

Reply:
Good catch, thanks!